src/mono/mono/component/mini-wasm-debugger.c

```diff
@@ -382,7 +382,7 @@ mono_wasm_send_dbg_command_with_parms (int id, MdbgProtCommandSet command_set, i
 		goto done;
 	}
 	MdbgProtBuffer bufWithParms;
-	buffer_init (&bufWithParms, 128);
+	m_dbgprot_buffer_init (&bufWithParms, 128);
```

Comment:
`debugger-engine.h` has `#define buffer_init m_dbgprot_buffer_init`. Is that not applicable here?

Reply:
yeah, I just used the full name because we also use m_dbgprot_buffer_add_int and m_dbgprot_buffer_add_byte_array.